# Abstract

<font color='blue' size='small'>
    Verbatim abstract text in blue follows each analysis.<br>
    </font>
<br>
SUBMITTED Version of ICPE 2020 KY Survey Early Prescribers (2339 characters of 2350 limit)<br>
<br>

---

**Do Early Prescribers Of New Drugs Have Different Risk Management Practices?**<br>
Submitted ICPE 2020 abstract #4157<br>
<br>
Dasgupta N, Brown JR, Freeman P, Nocera M, Slavova S<br>
<br>
## Background
Uptake of new technology occurs first in "early adopters." We hypothesized that early prescribers of new medicines could likewise be identified, possibly influencing interpretation of early post-market assessments.
<br>
## Objectives
Conduct a statewide prescriber survey to evaluate if self-described early prescribers have differences in risk management practices compared to later prescribers.
<br>
## Methods
Planned hypothesis generating analysis of a larger survey on opioid prescribing, fielded by state licensing authority in Nov 2019.
<br>**Setting**: All DEA-registered Kentucky (US) physicians.
<br>**Exposure**: Self-reported affirmation of any of 3 statements: I prescribe new medications before others; I enjoy the variety of prescribing new medicines; I like to share with colleagues about new medicines I've prescribed. (Other options: I feel more comfortable using familiar medications, etc.). <br>**Outcome**: Physician characteristics, opioid prescribing.
<br>**Analysis**: Descriptive univariable/bivariable stats
<br>
## Results
Emails were delivered to 7631 physicians, with 651 respondents (8.5% - similar to other unincentivized physician surveys). After limiting to controlled substance prescribers and removing incommpletes, the analysis sample was n=349, with 83 (24%) early prescribers. Early adopters were disproportionately in early or late career (less than 15 or 35+ practice-years). The male:female ratio was 2:1 among respondents, but proportionately more females (26%) than males (19%) were early adopters. There were no differences in patient load, practice setting, or medical specialty except emergency medicine and general surgery were less likely; and oncology and OB/GYN were more likely to be early adopters.<br>

Early prescribers were more likely to use opioid risk stratification tools (OR 1.5; 0.85, 2.7). Abuse deterrent formulation (ADF) opioid prescribing was similar (OR 1.1; 0.65, 1.8). However, early adopters were more likely (OR 1.4; 0.73, 2.6) to prescribe newer non-OxyContin ADFs, with 93% endorsing "innovative nature of abuse-deterrence mechanisms" as a consideration. They supported legislation mandating insurance coverage for ADFs (OR 3.5; 1.6, 8.9), and more strongly endorsed technical solutions like electronic prescription monitoring programs and urine drug screens, while opposing prescribing limits.
<br>
## Conclusions
In addition to prescribing newer medications, early adopters may differ in other healthcare delivery aspects that may impact care. The survey is being replicated in other states. Quantitative assessments will follow. Studies with historical controls may want to consider impacts of early adopters.

---
---
# Data preparation
## Import data, create ADF variables

In [75]:
display "Notebook generated on $S_DATE at $S_TIME ET"

cd "/Users/nabarun/Dropbox/Projects/P1 Surveys"
clear all
set more off

frame create pharm
    frame pharm: use pharmacistky
frame create docs
    frame docs: use prescriberky
frame dir
frame change docs

di "Exclude non-controlled substance prescribers and incomplete surveys:"
keep if contr_sub==1
    * n=206 deleted

// Create composite variables for ADF prescribing

local adfs "embeda hysinglaer morphabonder xtampzaer oxycontin"
foreach i of local adfs {
    gen any`i'=0
        replace any`i'=1 if inlist(`i',2,3,4,5)
            order any`i', a(`i')
                *tab any`i'
                    la var any`i' "Any reported prescribing of `i'"
}

gen anyadf=0
    replace anyadf=1 if anyembeda==1 | anyhysinglaer==1 | anymorphabond==1 | anyxtampzaer==1 | anyoxycontin==1
        la var anyadf "Any ADF prescribing"
            tab anyadf

gen anynotocadf=0
    replace anynotocadf=1 if anyembeda==1 | anyhysinglaer==1 | anymorphabond==1 | anyxtampzaer==1
        la var anynotocadf "Any non-OxyContin ADF prescribing"
            tab anynotocadf
            
// Recode into innovation influence as dichotmous
gen innovinfluence = 0
    replace innovinfluence = 1 if inlist(innovate,3,4)
        replace innovinfluence=. if innovate==.


Notebook generated on 12 Feb 2020 at 11:54:31 ET

/Users/nabarun/Dropbox/Projects/P1 Surveys




(MethodologicalAdvanc_DATA_NOHDRS_2020-01-28_1653.csv)


(MethodologicalAdvanc_DATA_NOHDRS_2020-01-28_1510.csv)

  default  0 x 0
  docs     686 x 105; MethodologicalAdvanc_DATA_NOHDRS_2020-01-28_1510.csv
  pharm    827 x 107; MethodologicalAdvanc_DATA_NOHDRS_2020-01-28_1653.csv


Exclude non-controlled substance prescribers and incomplete surveys:

(206 observations deleted)


(46 real changes made)
(37 real changes made)
(54 real changes made)
(44 real changes made)
(176 real changes made)


(187 real changes made)



    Any ADF |
prescribing |      Freq.     Percent        Cum.
------------+-----------------------------------
          0 |        293       61.04       61.04
          1 |        187       38.96      100.00
------------+-----------------------------------
      Total |        480      100.00


(71 real changes made)



        Any |
non-OxyCont |
     in ADF |
prescribin

---
## Early adopter distributions

In [76]:
la var early_adopt___1 "Usually prescribe new medications before others do"
la var early_adopt___2 "Prefer medications which have worked well for patients in the past"
la var early_adopt___3 "Like being able to share with colleagues about new medications I've prescribed"
la var early_adopt___4 "Like the variety of prescribing new medications"
la var early_adopt___5 "Feel more comfortable using familiar medications"
la var early_adopt___6 "Prefer to wait until I hear about colleagues' experiences with prescribing new medications"
la var early_adopt___7 "Other"

foreach var of varlist early_adopt___1-early_adopt___6 {
tab `var',
}
set linesize 90
list early_adopt_other if early_adopt_other!=""







note: label truncated to 80 characters



    Usually |
  prescribe |
        new |
medications |
     before |
  others do |      Freq.     Percent        Cum.
------------+-----------------------------------
  Unchecked |        461       96.04       96.04
    Checked |         19        3.96      100.00
------------+-----------------------------------
      Total |        480      100.00

     Prefer |
medications |
 which have |
worked well |
        for |
patients in |
   the past |      Freq.     Percent        Cum.
------------+-----------------------------------
  Unchecked |        315       65.62       65.62
    Checked |        165       34.38      100.00
------------+-----------------------------------
      Total |        480      100.00

 Like being |
    able to |
 share with |
 colleagues |
  about new |
medications |
       I've |
 prescribed |      Freq.     Percent        Cum.
------------+-----------------------------------
  Unchecked |        444       92.50

## Define early prescribers
Self-reported affirmative response to any of 3 statements:<br>
`early_adopt___1`   prescribing new medications before others<br>
`early_adopt___3`   enjoying variety of prescribing new medicines<br>
`early_adopt___4`   liking to share with colleagues about new medicines

In [77]:
// Create variables for early adopters
gen earliest=0
    la var earliest "Self-identified early adopters"
        replace earliest = 1 if early_adopt___1==1
            tab earliest

gen early=0
    la var early "Enjoys novel prescribing"
        replace early = 1 if early_adopt___1==1 | early_adopt___3==1 | early_adopt___4==1
            tab early




(19 real changes made)


Self-identi |
 fied early |
   adopters |      Freq.     Percent        Cum.
------------+-----------------------------------
          0 |        461       96.04       96.04
          1 |         19        3.96      100.00
------------+-----------------------------------
      Total |        480      100.00



(83 real changes made)


     Enjoys |
      novel |
prescribing |      Freq.     Percent        Cum.
------------+-----------------------------------
          0 |        397       82.71       82.71
          1 |         83       17.29      100.00
------------+-----------------------------------
      Total |        480      100.00


## Exclude incomplete surveys
Keep if survey is eventually incomplete but respondent had self-identified as an early prescriber or not.

In [78]:
di "Exclude incomplete surveys while retaining early prescribers (see Section 5 for rationale):"
keep if physician_survey_complete==2 | early==1
    * n=131 should be deleted

tab earliest
tab early


Exclude incomplete surveys while retaining early prescribers (see Section 5 for rationale)
> :

(131 observations deleted)


Self-identi |
 fied early |
   adopters |      Freq.     Percent        Cum.
------------+-----------------------------------
          0 |        330       94.56       94.56
          1 |         19        5.44      100.00
------------+-----------------------------------
      Total |        349      100.00


     Enjoys |
      novel |
prescribing |      Freq.     Percent        Cum.
------------+-----------------------------------
          0 |        266       76.22       76.22
          1 |         83       23.78      100.00
------------+-----------------------------------
      Total |        349      100.00


---
# Physician Characteristics
## Gender

In [79]:
tab gender early_adopt___1, col nokey
tab gender early_adopt___3, col nokey
tab gender early_adopt___4, col nokey

tab gender earliest, col nokey

tab gender early, col nokey
tab gender early, row nokey



                      | Usually prescribe new
                      |  medications before
                      |       others do
 What is your gender? | Unchecked    Checked |     Total
----------------------+----------------------+----------
                 Male |       207         14 |       221 
                      |     64.89      77.78 |     65.58 
----------------------+----------------------+----------
               Female |       107          4 |       111 
                      |     33.54      22.22 |     32.94 
----------------------+----------------------+----------
 Prefer not to answer |         5          0 |         5 
                      |      1.57       0.00 |      1.48 
----------------------+----------------------+----------
                Total |       319         18 |       337 
                      |    100.00     100.00 |    100.00 


                      |  Like being able to
                      | share with colleagues
                      | abo

<br>
<font color='blue'>
The male:female ratio was 2:1 among respondents, but proportionately more females (26%) than males (19%) were early adopters.
    </font>

---
## Years in practice

In [80]:
di "Among non-early prescribers:"
tab years_prac if early==0, p 
di "Among early prescribers:"
tab years_prac if early==1, p

tab years_prac early, col nokey
ranksum years_prac, by(early)
tab years_prac earliest, col nokey
ranksum years_prac, by(earliest)


Among non-early prescribers:


     Please indicate your total |
    number of years in practice |
      following completion of r |      Freq.
--------------------------------+------------+---------------------------------
                            < 5 |         30 |*************
                          5 -15 |         55 |************************
                        16 - 25 |         77 |*********************************
                        26 - 35 |         59 |*************************
                           > 35 |         44 |*******************
--------------------------------+------------+---------------------------------
                          Total |        265 

Among early prescribers:


     Please indicate your total |
    number of years in practice |
      following completion of r |      Freq.
--------------------------------+------------+---------------------------------
                            < 5 |         13 |*************
                   

There is some murkiness here and low cell sizes, but the trend is worth keeping an eye on.
<br>
<font color='blue'>
Early prescribers tended to be disproportionately in early or late career (<15 and 35+ practice-years, respectively).
    </font>
    <br>

---
## Number of patients

In [81]:
tab no_patients early, col nokey


        On |
  average, |
across all |
  practice |
     sites |
  (primary |
  practice |
 and other |     Enjoys novel
 practice) |      prescribing
       how |         0          1 |     Total
-----------+----------------------+----------
      < 25 |        33          5 |        38 
           |     12.45       6.94 |     11.28 
-----------+----------------------+----------
     25-49 |        54         17 |        71 
           |     20.38      23.61 |     21.07 
-----------+----------------------+----------
     50-74 |        69         18 |        87 
           |     26.04      25.00 |     25.82 
-----------+----------------------+----------
    75-100 |        73         20 |        93 
           |     27.55      27.78 |     27.60 
-----------+----------------------+----------
     > 100 |        36         12 |        48 
           |     13.58      16.67 |     14.24 
-----------+----------------------+----------
     Total |       265         72 |       337 
         

<font color='blue'>
There was no difference in number of patients seen per week.</font>

---
## Specialty

In [82]:
tab specialty_type early, col nokey


         Which of the |
       following best |     Enjoys novel
       describes your |      prescribing
  primary specialty?  |         0          1 |     Total
----------------------+----------------------+----------
   Addiction medicine |         2          4 |         6 
                      |      0.75       5.56 |      1.78 
----------------------+----------------------+----------
       Anesthesiology |         6          0 |         6 
                      |      2.26       0.00 |      1.78 
----------------------+----------------------+----------
Cardiovascular diseas |         3          1 |         4 
                      |      1.13       1.39 |      1.19 
----------------------+----------------------+----------
Critical care medicin |         2          0 |         2 
                      |      0.75       0.00 |      0.59 
----------------------+----------------------+----------
   Emergency medicine |        42          6 |        48 
                      |     1

<font color='blue'>
There were no significant differences in medical specialty... possibly except emergency medicine and general surgery less likely, and hem/onc and OB/GYN more likely to be early adopters, but cell sizes were small.</font>

---
## Practice setting

In [83]:
tab prac_type early, col nokey


         Which of the |
       following best |
   characterizes your |     Enjoys novel
     primary practice |      prescribing
             setting? |         0          1 |     Total
----------------------+----------------------+----------
        Solo practice |        28          9 |        37 
                      |     10.61      12.50 |     11.01 
----------------------+----------------------+----------
Small private group p |        33          9 |        42 
                      |     12.50      12.50 |     12.50 
----------------------+----------------------+----------
Large private group p |        32         14 |        46 
                      |     12.12      19.44 |     13.69 
----------------------+----------------------+----------
    Academic practice |        36          9 |        45 
                      |     13.64      12.50 |     13.39 
----------------------+----------------------+----------
 Emergency department |        43          7 |        50 
     

<font color='blue'>
Likewise, there were no appreciable differences in practice setting, possibly with the exception of ED physicians less likely to prescribe new medicines.</font>

---
# Opioid Prescribing
## Risk stratification

In [84]:
tab riskstrat early, m col nokey
    gen risktool = 1
        replace risktool=0 if riskstrat==.
            cc risktool early



         Which of the |
            following |
  risk-stratification |     Enjoys novel
    tools do you most |      prescribing
   commonly utilize?  |         0          1 |     Total
----------------------+----------------------+----------
     Opioid risk tool |        27          7 |        34 
                      |     10.15       8.43 |      9.74 
----------------------+----------------------+----------
Screener and Opioid A |         4          2 |         6 
                      |      1.50       2.41 |      1.72 
----------------------+----------------------+----------
Screener and Opioid A |         7          6 |        13 
                      |      2.63       7.23 |      3.72 
----------------------+----------------------+----------
 Brief Risk Interview |        23         11 |        34 
                      |      8.65      13.25 |      9.74 
----------------------+----------------------+----------
                Other |         3          1 |         4 
    

<font color='blue'>
Early prescribers were more likely to use opioid risk stratification tools (32% vs. 24%). Early prescribers were more likely to use opioid risk stratification tools (OR 1.5; 0.85, 2.7). </font>

---

## ADFs

In [85]:
tab anyadf early, col nokey
cc anyadf early



   Any ADF |     Enjoys novel
prescribin |      prescribing
         g |         0          1 |     Total
-----------+----------------------+----------
         0 |       131         39 |       170 
           |     49.25      46.99 |     48.71 
-----------+----------------------+----------
         1 |       135         44 |       179 
           |     50.75      53.01 |     51.29 
-----------+----------------------+----------
     Total |       266         83 |       349 
           |    100.00     100.00 |    100.00 

                                                         Proportion
                 |   Exposed   Unexposed  |      Total     Exposed
-----------------+------------------------+------------------------
           Cases |        44         135  |        179       0.2458
        Controls |        39         131  |        170       0.2294
-----------------+------------------------+------------------------
           Total |        83         266  |        349       0.2

In [86]:
tab anynotocadf early, col nokey
cc anynotocadf early



       Any |
non-OxyCon |
   tin ADF |     Enjoys novel
prescribin |      prescribing
         g |         0          1 |     Total
-----------+----------------------+----------
         0 |       217         63 |       280 
           |     81.58      75.90 |     80.23 
-----------+----------------------+----------
         1 |        49         20 |        69 
           |     18.42      24.10 |     19.77 
-----------+----------------------+----------
     Total |       266         83 |       349 
           |    100.00     100.00 |    100.00 

                                                         Proportion
                 |   Exposed   Unexposed  |      Total     Exposed
-----------------+------------------------+------------------------
           Cases |        20          49  |         69       0.2899
        Controls |        63         217  |        280       0.2250
-----------------+------------------------+------------------------
           Total |        83         2

---
## Innovation

In [87]:
tab innovate early, m col nokey
    ranksum innovate if innovate!=., by(early)
tab innovinfluence early, col nokey



      Innovative |
       nature of |     Enjoys novel
abuse-deterrence |      prescribing
      mechanisms |         0          1 |     Total
-----------------+----------------------+----------
    No influence |        12          1 |        13 
                 |      4.51       1.20 |      3.72 
-----------------+----------------------+----------
Little influence |        32          1 |        33 
                 |     12.03       1.20 |      9.46 
-----------------+----------------------+----------
  Some influence |        32         23 |        55 
                 |     12.03      27.71 |     15.76 
-----------------+----------------------+----------
 Great influence |         5          5 |        10 
                 |      1.88       6.02 |      2.87 
-----------------+----------------------+----------
               . |       185         53 |       238 
                 |     69.55      63.86 |     68.19 
-----------------+----------------------+----------
           To

<font color='blue'>
Early and standard prescribers had similar abuse deterrent formulation (ADF) opioid prescribing (OR 1.1; 0.6, 1.8). However, early adopters were more likely (OR 1.4; 0.7, 2.6) to prescribe newer non-OxyContin ADFs, with 93% endorsing "innovative nature of abuse-deterrence mechanisms" as a consideration.</font>
<br>
<br>

---

## Legislation, PMP, limits

In [88]:
tab state_legis early, col nokey
mdesc state_legis
cc state_legis early



   At this |
     time, |
 would you |
   support |
     state |
legislatio |
         n |
 mandating |     Enjoys novel
third-part |      prescribing
y payer co |         0          1 |     Total
-----------+----------------------+----------
        No |        78          8 |        86 
           |     30.23      10.96 |     25.98 
-----------+----------------------+----------
       Yes |       180         65 |       245 
           |     69.77      89.04 |     74.02 
-----------+----------------------+----------
     Total |       258         73 |       331 
           |    100.00     100.00 |    100.00 


    Variable    |     Missing          Total     Percent Missing
----------------+-----------------------------------------------
    state_legis |          18            349           5.16
----------------+-----------------------------------------------

                                                         Proportion
                 |   Exposed   Unexposed  |      Total 

In [89]:
tab dslimits early, col nokey m
    ranksum dslimits if dslimits!=5, by(early)



                     |     Enjoys novel
   Prescribing (days |      prescribing
      supply) limits |         0          1 |     Total
---------------------+----------------------+----------
Not effective at all |        18          9 |        27 
                     |      6.77      10.84 |      7.74 
---------------------+----------------------+----------
  Somewhat effective |        81         18 |        99 
                     |     30.45      21.69 |     28.37 
---------------------+----------------------+----------
           Effective |        99         26 |       125 
                     |     37.22      31.33 |     35.82 
---------------------+----------------------+----------
      Very effective |        59         14 |        73 
                     |     22.18      16.87 |     20.92 
---------------------+----------------------+----------
              Unsure |         9          4 |        13 
                     |      3.38       4.82 |      3.72 
------------

In [90]:
tab urineds early, col nokey m
    ranksum urineds if urineds!=5, by(early)



                     |     Enjoys novel
                     |      prescribing
Urine drug screening |         0          1 |     Total
---------------------+----------------------+----------
Not effective at all |        22          4 |        26 
                     |      8.27       4.82 |      7.45 
---------------------+----------------------+----------
  Somewhat effective |        89         24 |       113 
                     |     33.46      28.92 |     32.38 
---------------------+----------------------+----------
           Effective |        99         15 |       114 
                     |     37.22      18.07 |     32.66 
---------------------+----------------------+----------
      Very effective |        42         23 |        65 
                     |     15.79      27.71 |     18.62 
---------------------+----------------------+----------
              Unsure |        14          6 |        20 
                     |      5.26       7.23 |      5.73 
------------

In [91]:
tab kasper early, col nokey m
    ranksum kasper if kasper!=5, by(early)



        Checking the |
   prescription drug |     Enjoys novel
  monitoring program |      prescribing
            (KASPER) |         0          1 |     Total
---------------------+----------------------+----------
Not effective at all |         6          0 |         6 
                     |      2.26       0.00 |      1.72 
---------------------+----------------------+----------
  Somewhat effective |        65         12 |        77 
                     |     24.44      14.46 |     22.06 
---------------------+----------------------+----------
           Effective |        91         27 |       118 
                     |     34.21      32.53 |     33.81 
---------------------+----------------------+----------
      Very effective |       100         31 |       131 
                     |     37.59      37.35 |     37.54 
---------------------+----------------------+----------
              Unsure |         4          1 |         5 
                     |      1.50       1.20 | 

<font color='blue'>
They were more likely to support legislation mandating insurance coverage for ADFs (OR 1.3), and more strongly endorsed technical solutions like electronic prescription monitoring programs and urine drug screens, while more vehemently opposing prescribing limits. </font>

---
# Survey completion sub-analysis
Objective: To assess furthest question responded to as a marker for inclusion. This was deemed important because some respondents clicked on no more than the initial screen.

In [92]:
// Early adopter analysis for IPCE abstract
//

cd "/Users/nabarun/Dropbox/Projects/P1 Surveys"
clear all
set more off

use prescriberky

di "Exclude non-controlled substance prescribers and incomplete surveys:"
keep if contr_sub==1
distinct record_id

di "Check within early adopters:"
gen early=0
    la var early "Enjoys novel prescribing"
        replace early = 1 if early_adopt___1==1 | early_adopt___3==1 | early_adopt___4==1
            tab early

foreach x of varlist _all {
rename `x' zz`x'
} 
rename zz* v#, renumber
    rename v1 id
        rename v106 early
            gen incomplete=regexm(lower(v3),"not completed")
                di "Drop free text strings:""
                    drop v2 v4 v3 v12 v13 v21 v33 v63 v65 v73 v81 v82 v98 v101 v104

reshape long v, i(id)
    drop if v==. & _j!=2
        drop if inlist(v,0,1,2)

bysort id: egen last = max(_j) 
    drop _j v 

duplicates drop

di "Describe length of completeness among incomplete surveys:"
tab last if incomplete==1

di "Look at distribution within early prescribers:"
tab last if incomplete==1 & early==1



/Users/nabarun/Dropbox/Projects/P1 Surveys



(MethodologicalAdvanc_DATA_NOHDRS_2020-01-28_1510.csv)

Exclude non-controlled substance prescribers and incomplete surveys:

(206 observations deleted)


           |        Observations
           |      total   distinct
-----------+----------------------
 record_id |        480        480

Check within early adopters:



(83 real changes made)


     Enjoys |
      novel |
prescribing |      Freq.     Percent        Cum.
------------+-----------------------------------
          0 |        397       82.71       82.71
          1 |         83       17.29      100.00
------------+-----------------------------------
      Total |        480      100.00






Drop free text strings:


(note: j = 5 6 7 8 9 10 11 14 15 16 17 18 19 20 22 23 24 25 26 27 28 29 30 31 32 34 35 36 
> 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 64 66 67 6
> 8 69 70 71 72 74 75 76 77 78 79 80 83 84 85 86 87 88 89 90 91 92 93 94 95 96

## Interpretation
About 70% of incomplete surveys respondents stopped after about the 11th question, which was the radio button question about familiarity with  ADF formulations and route of preventing abuse. This suggests a general unwillingness to engage in the route of exposure question, despite this being a key element of labeling.<br>
<br>
The analytic implication is to not limit by overall incomplete surveys, but rather to limit to those who we have a determiniation of early adopter status. Incomplete responses to specific later survey questions are to be treated as missing. Among early prescribers, there doesn't appear to be much of a pattern...5 or so did not complete beyond the familiarity/route table. Since the early adopter question construction relies on negative data (e.g., boxes unchecked, the handling of missingness can be differential to reflect the variable construction definition.<br>
<br>
The analysis above has been updated to refelct this design decision. There are 2 final exclusion criteria:<br><br> First, if limiting to controlled substance prescribers, removing 206 observations out of the original 686 responses received:
`keep if contr_sub==1`
<br><br>
Second, remove incomplete surveys but retain surveys among early prescribers, removing 131 observations for an analysis set of 349.<br>
`keep if physician_survey_complete==2 | early==1`
<br><br>
In the next iteration of the survey we should think about moving that radio button table or simplifying it.<br><br>

---
# Future considerations
Replication in other states will be helpful to have more sample size. Further characterization of early adopters could require more or better questions. Using the `earliest` self-identified category could serve as a sub-group analysis for a specificity and strength gradient. I wonder if there are measures of sociality (professional connectedness) that might explain some of the tendency to prescribe early, e.g., network effect for incoming information about new drugs or social milleu where having conversations about new drugs is rewarding.

---
*fin.*